ludwig/contribs/mlflow/__init__.py

```diff
@@ -171,16 +174,50 @@ def on_visualize_figure(self, fig):
         pass
 
     def prepare_ray_tune(self, train_fn, tune_config, tune_callbacks):
-        from ray.tune.integration.mlflow import mlflow_mixin
-
-        return mlflow_mixin(train_fn), {
-            **tune_config,
-            "mlflow": {
-                "experiment_id": self.experiment_id,
-                "experiment_name": self.experiment_name,
-                "tracking_uri": mlflow.get_tracking_uri(),
-            },
-        }
+        import ray
+
+        if version.parse(ray.__version__) >= version.parse("2.3.0"):
+
```

Comment:
Is `2.3.0` the right min version? It was working before with 2.3, right? So maybe we just want to check against 2.4?

Reply:
Will do. The new mlflow integration also works with 2.3, but we can introduce it with the 2.4 bump.